### PR TITLE
Dicom CI Overall Test

### PR DESCRIPTION
**Pull request by QuanWanxx**
1. Create the Overall test for DicomToDatalake pipeline.
2. Change **version** to **apiVersion** for DicomServer data source.

### CODE REVIEW

File: FhirToDataLake/synapse/DicomToSynapseValidator.py [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+import requests
+import pyodbc
+import pandas as pd
+import time
+
+parser = argparse.ArgumentParser()
+parser.add_argument("--synapse_workspace", required=True, help="Name of Synapse workspace.")
+parser.add_argument("--dicom_server_url", required=True, help="Dicom server url.")
+parser.add_argument("--dicom_server_access_token", help="Dicom server bearer access token.")
+parser.add_argument("--schema_directory", required=True, help="Schema directory path.")
```
Comment (BoyaWu10) [inline FhirToDataLake/synapse/DicomToSynapseValidator.py:14]: Would it be convenient to add customized schema testing for DICOM as well?

File: FhirToDataLake/azurepipelines/OverallTest/DeployContainerInstance.json [hunk 2]
```diff
@@ -0,0 +1,405 @@
+{
```
Comment (BoyaWu10) [inline FhirToDataLake/azurepipelines/OverallTest/DeployContainerInstance.json:1]: Is this template used in overall test?

File: FhirToDataLake/synapse/DicomToSynapseValidator.py (hunk 1 above)
Comment (QuanWanxx) [reply FhirToDataLake/synapse/DicomToSynapseValidator.py:15]: Added customized schema test.

File: FhirToDataLake/azurepipelines/OverallTest/DeployContainerInstance.json (hunk 2 above)
Comment (QuanWanxx) [reply FhirToDataLake/azurepipelines/OverallTest/DeployContainerInstance.json:1]: No, the overall test uses Container Instance that been deployed be this template, I just want to add this for reference.

Comment (BoyaWu10) [reply FhirToDataLake/azurepipelines/OverallTest/DeployContainerInstance.json:1]: Got it. Thanks!

**Review by BoyaWu10 — APPROVED** [review]

**Pull request merged**